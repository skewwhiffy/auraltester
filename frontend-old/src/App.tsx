import {RouterProvider} from 'react-router-dom'
import {createBrowserRouter} from 'react-router-dom'
import routesConfig from "./application/routesConfig";
import './index.css'
import {QueryClient, QueryClientProvider} from "@tanstack/react-query";

function App() {
  const router = createBrowserRouter(routesConfig);
  const queryClient = new QueryClient()
  return (
    <QueryClientProvider client={queryClient}>
      <RouterProvider router={router}/>
    </QueryClientProvider>
  )
}

export default App
