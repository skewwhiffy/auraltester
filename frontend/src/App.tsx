import React from 'react'
import Scale from './component/Scale'
import Interval from './component/Interval'
import NavBar from './component/NavBar'
import { BrowserRouter, Route, Routes } from 'react-router-dom'
import 'bootstrap/dist/css/bootstrap.min.css';

class App extends React.Component {
  render() {
    return (
      <div className="App">
        <BrowserRouter>
          <NavBar />
          <Routes>
            <Route index element={<p>Hello</p>} />
            <Route path="scales" element={<Scale />} />
            <Route path="intervals" element={<Interval />} />
          </Routes>
        </BrowserRouter>
      </div>
    )
  }
}

export default App
