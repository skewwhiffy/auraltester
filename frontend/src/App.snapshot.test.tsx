import renderer from 'react-test-renderer'
import App from './App'

jest.mock('./component/Scale', () => () => (<div>SCALE</div>))
jest.mock('./component/Interval', () => () => (<div>INTERVAL</div>))
jest.mock('./component/NavBar', () => () => (<div>NAVBAR</div>))
describe('App snapshot', () => {
  it('should match snapshot', () => {
    const actual = renderer.create(
      <App/>
    )
    
    expect(actual).toMatchSnapshot()
  })
})