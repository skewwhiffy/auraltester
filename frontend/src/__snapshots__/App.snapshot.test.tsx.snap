// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot should match snapshot 1`] = `
<div
  className="App"
>
  <div>
    NAVBAR
  </div>
  <p>
    Hello
  </p>
</div>
`;
